### ISSUE TITLE
Is it working under ruby 2.0.0?

### ISSUE THREAD

**Issue by jmarceli** [astashov/vim-ruby-debugger#94]
When I try to start the debugger (`:Rdebugger` in Gvim) I get message:

```
Debugger started
```

But in ruby_debugger_output there is a message:

```
Uncaught exception: cannot load such file -- /path/to/my/app/script/rails
```

And debugger is not working at all. I use ruby 2.0.0 is it a problem, maybe there is some easy solution?

**Comment by runephilosof** [comment]
vim-ruby-debugger uses the debugger-xml gem https://github.com/astashov/debugger-xml which uses the debugger gem, which doesn't work for 2.x https://github.com/cldwalker/debugger/issues/125
Actually you should have seen an error when you tried to install the debugger-xml gem, since it would fail at installing the debugger gem.

**Comment by deivid-rodriguez** [comment]
Support for Ruby 2.x is coming soon guys!

**Comment by jmarceli** [comment]
It would be really nice, tell me if I can help somehow (any branch or repo with dev version?). I'm not an expert but I will try to do my best.

**Comment by deivid-rodriguez** [comment]
@jmarceli Thanks for the offer. @astashov knows it best but as far as I know, `vim-ruby-debugger` doesn't need any changes. It should be enough that it uses latest `debugger-xml` and latest `byebug` (when it is released).

**Comment by astashov** [comment]
Yup, @deivid-rodriguez is right. Once the next version of `byebug` is released, I'll add its support to `debugger-xml` (there is already a PR for that), and you just will need to specify the newer version in your Gemfile to make it work.

**Comment by jmarceli** [comment]
OK. Thanks for the info. Now I'm using `pry` but it would be nice to have debugger integrated with vim.
